### DISCUSSION TITLE
Rust server is not working

### DISCUSSION THREAD

**Discussion by pompsa** [GameServerManagers/LinuxGSM#3528] [General]
Hello, my rust server is not working.. Here is it from console:

08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/libkernel32
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/libkernel32.so
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/kernel32
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/libkernel32
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/libkernel32.so
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/libkernel32
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/data-0x5600700eb3f0.so
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/data-0x56007016a8d0.so
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/data-0x560070142660.so
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/data-0x5600701d3130.so
08:36:46 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/data-0x5600701e19c0.so
08:36:50 Fallback handler could not load library /hlserver/27843/RustDedicated_Data/Mono/data-0x560070f73d20.so

Please help, what i have to do?

**Comment by h3o66** [comment]
Hi,
please provide more info to be able to help you.
We cannot really support someone who does not give any info.
As alternative join the lgsm discord and ask for help.
